test/builtins/ThresholdPredictors.jl

```diff
@@ -268,6 +268,61 @@ MMI.input_scitype(::Type{<:NaiveClassifier}) = Table(Continuous)
         mode(Distributions.fit(MLJBase.UnivariateFinite, y[I])), MLJBase.nrows(X)
     )
 end
+
+
+# define a probabilistic classifier with non-persistent `fitresult`, but which addresses
+# this by overloading `save`/`restore`:
+thing = []
+struct EphemeralClassifier <: MLJBase.Probabilistic end
+function MLJBase.fit(::EphemeralClassifier, verbosity, X, y)
+    # if I serialize/deserialized `thing` then `view` below changes:
+    view = objectid(thing)
```

Comment:
can we use a different variable name? Then we should add in the code comments that  `object_id` is the object's hash value. Which changes when the object is serialized.